src/rust/ensogl/lib/theme/src/lib.rs

```diff
@@ -275,7 +285,7 @@ define_themes! { [light:0, dark:1]
             highlight  = selection , selection;
             text = Lcha(0.0,0.0,0.0,0.7) , Lcha(1.0,0.0,0.0,0.7);
             text {
-                highlight = Lcha(0.8,0.0,0.0,1.0) , Lcha(0.7,0.0,0.0,1.0);
+                highlight = Lcha(0.7,0.4,0.74,1.0) , Lcha(0.7,0.4,0.74,1.0);
```

Comment:
do not hardcode values, link them to other places in the stylesheet if possible. (see my previous comment)

Reply:
I don't think we have a matching value here. This is for the blue highlight in the searcher.